plugin/al/mayautils/AL/maya/utils/PluginTranslatorOptions.h

```diff
@@ -387,6 +410,9 @@ class PluginTranslatorOptions
         MString              defString;   ///< default string value
         std::vector<MString> enumStrings; ///< the text values for the enums
         OptionType           type;        ///< the type of the option
+        int                  precision { 1 };
+        std::string          controller {};
```

Comment:
Same here: I'd use a MString instead of std::string.

Reply:
Updated.